knimin/lib/util.py

```diff
@@ -232,16 +246,21 @@ def categorize_etoh(x):
 
 
 def categorize_bmi(x):
-    if x < 0:
+    if x == 'Unspecified':
+        return 'Unspecified'
```

Comment:
The issue is it comes from the conversions, so it can be a float or 'Unspecified' if it can't be calculated, thus this test is needed.